packages/account/src/Configs/proof-of-identity-config.ts

```diff
@@ -1,20 +0,0 @@
-import React from 'react';
```

Comment:
why is it deleted?

Reply:
It is dead code. Thanks. I've added a checklist to verify